ui/src/app/cohort-search/constant.ts

```diff
@@ -19,3 +19,13 @@ export const DOMAIN_TYPES = [
     // { name: 'Vitals',       type: 'vitals' },
     // { name: 'Temporal',     type: 'temporal' }
 ];
```

Comment:
Would it make sense to add a property here for each domain type that can be used for each type in the code base instead of using Strings? For example: 'PM'?

Reply:
Yes, that makes sense. I'll create a new story for that.